tests/safety/common.py

```diff
@@ -562,6 +563,10 @@ def _reset_angle_measurement(self, angle):
     for _ in range(6):
       self._rx(self._angle_meas_msg(angle))
 
+  def _reset_speed_measurement(self, speed):
+    for _ in range(6):
+      self._rx(self._speed_msg(speed))  # pylint: disable=no-member
```

Comment:
why is the pylint exception needed?

Reply:
This test is done in the angle safety as no other safety mode needs to set the vehicle speed,  `_speed_msg` is defined as abstract in the base panda safety test. However, it shouldn't violate anything to also define _speed_msg as abstract in the angle safety class so the dependency is explicit